### PR TITLE
DX-1301: Bug fix to create collection example and also implemented Ge…

### PR DESCRIPTION
**Pull request by suluvai**
# Summary
Bug fix to create collection example and also implemented GetPublicKey for signers [DX-1301](https://immutable.atlassian.net/browse/DX-1301)

# Why the changes
https://github.com/immutable/imx-core-sdk-golang/issues/81

### CODE REVIEW

File: imx/signers/stark/signer.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func (base *Signer) VerifySignature(hash *big.Int, signature, signersPublicKey s
 }
 
 // GetAddress returns the stark public key of the StarkSigner as a 64 digit hex string prefixed with 0x.
```
Comment (zatlite) [inline imx/signers/stark/signer.go:86]: Are we not keeping these method names in sync with the core-sdk-ts?

Comment (zatlite) [reply imx/signers/stark/signer.go:86]: Also, if this method name is changed it should also change on the godoc comment above.

Comment (suluvai) [reply imx/signers/stark/signer.go:86]: Yup, godoc needs update. Typescript SDK will need to be updated as well.

Comment (suluvai) [reply imx/signers/stark/signer.go]: Created https://immutable.atlassian.net/browse/DX-1310 to address Core SDK TS

**Review by zatlite — APPROVED** [review]

**Pull request merged**